sass/contact.scss

```diff
@@ -0,0 +1,83 @@
+@import '_variables';
+@import '_mixins';
+
+.contact{
+  padding-bottom: 40px;
+  .details_map {
+    .details {
+      padding: 0;
+      margin: 0;
+      list-style: none;
+      span {
+        color: $tertiary_color;
+        @include font-size(18px);
+        font-weight: 400;
+        line-height: 27px;
+        letter-spacing: 0px;
+        padding-left: 11px;
+      }
+      i {
+        color: $primary_color;
+        font-size: 1em;
+      }
+    }
+
+
```

Comment:
Remove these empty spaces. You can leave only one.

Reply:
![a7e](https://user-images.githubusercontent.com/41734742/61533153-a41e9600-aa34-11e9-9780-e6530b728093.jpg)

Reply:
:))))